### PR TITLE
Add data model for cabinetU

### PR DESCRIPTION
**Pull request by Pengpengwanga**
Signed-off-by: Pengpeng Wang <pengpengw@vmware.com>

### CODE REVIEW

File: flowgate-common/src/main/java/com/vmware/flowgate/common/FlowgateConstant.java [hunk 1]
```diff
@@ -64,6 +64,8 @@
 
    public static String SENSOR = "SENSOR";
 
+   public static String CABINETUS = "CabinetUs";
```
Comment (yixingjia) [inline flowgate-common/src/main/java/com/vmware/flowgate/common/FlowgateConstant.java:67]: CabinetUnits

File: flowgate-common/src/main/java/com/vmware/flowgate/common/model/CabinetU.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package com.vmware.flowgate.common.model;
+
+public class CabinetU {
+
+   private String state;
```
Comment (yixingjia) [inline flowgate-common/src/main/java/com/vmware/flowgate/common/model/CabinetU.java:5]: used

File: flowgate-common/src/main/java/com/vmware/flowgate/common/model/CabinetU.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+   private int cabinetAssetID;
+   private int cabinetUNumber;
+   private String assetsOnU;
```
Comment (yixingjia) [inline flowgate-common/src/main/java/com/vmware/flowgate/common/model/CabinetU.java:8]: assetOnUnit

File: flowgate-common/src/main/java/com/vmware/flowgate/common/FlowgateConstant.java (hunk 1 above)
Comment (Pengpengwanga) [reply flowgate-common/src/main/java/com/vmware/flowgate/common/FlowgateConstant.java]: done

File: flowgate-common/src/main/java/com/vmware/flowgate/common/model/CabinetU.java (hunk 2 above)
Comment (Pengpengwanga) [reply flowgate-common/src/main/java/com/vmware/flowgate/common/model/CabinetU.java]: Done

File: flowgate-common/src/main/java/com/vmware/flowgate/common/model/CabinetU.java (hunk 3 above)
Comment (Pengpengwanga) [reply flowgate-common/src/main/java/com/vmware/flowgate/common/model/CabinetU.java]: Done

**Pull request merged**